pkg/webhooks/openstackmachine_webhook.go

```diff
@@ -96,6 +96,7 @@ func (*openStackMachineWebhook) ValidateUpdate(_ context.Context, oldObjRaw, new
 		delete(newOpenStackMachineSpec, "providerID")
 	}
 
+	// instanceID is present in v1alpha6 but is deprecated in v1beta1
```

Comment:
This is the problem of writing these webhooks with `Unstructured` instead of proper types. The code below is a no-op because `instanceID` can never be set. If we were using proper types it would have been a compile error and we'd have noticed.

This can be deleted. Probably best in a separate PR, though, or at least a separate commit.

Reply:
`// instanceID is present in v1alpha6 but is deprecated in v1beta1` was removed